nginx-controller/controller/controller.go

```diff
@@ -287,10 +287,14 @@ func (lbc *LoadBalancerController) syncEndp(key string) {
 			if !isNginxIngress(&ing) {
 				continue
 			}
-			ingEx := lbc.createIngress(&ing)
+			ingEx, err := lbc.createIngress(&ing)
+			if err != nil {
+				glog.Warningf("Error updating endpoints for %v/%v: %v, skipping", ing.Namespace, ing.Name, err)
+				return
```

Comment:
a small bug. must be `continue`  instead of `return`. This leads to Ignoring the rest of the Ingresses.